base/src/EglRenderer.cpp

```diff
@@ -17,17 +18,17 @@ class EglRenderer::Detail
         }
     }
 
-    bool init(uint32_t _height, uint32_t _width){
+    bool init(uint32_t _height, uint32_t _width , bool _displayOnTop){
         uint32_t displayHeight, displayWidth;
         NvEglRenderer::getDisplayResolution(displayWidth,displayHeight);
         if(height!=0 && width!=0){
             x_offset += (displayWidth-width)/2;
             y_offset += (displayHeight-height)/2;
-            renderer = NvEglRenderer::createEglRenderer(__TIMESTAMP__, width, height, x_offset, y_offset);
+            renderer = NvEglRenderer::createEglRenderer(__TIMESTAMP__, width, height, x_offset, y_offset,displayOnTop);
         }else{
             x_offset += (displayWidth-_width)/2;
             y_offset += (displayHeight-_height)/2;
-            renderer = NvEglRenderer::createEglRenderer(__TIMESTAMP__, _width, _height, x_offset, y_offset);
+            renderer = NvEglRenderer::createEglRenderer(__TIMESTAMP__, _width, _height, x_offset, y_offset, _displayOnTop);
```

Comment:
use displayOnTop Instead of _displayOnTop

Reply:
changed